### PR TITLE
fix(test) docker logs stdout for failures

### PR DESCRIPTION
**Pull request by dshekhar95**
## Problem
Lack of visibility of Docker logs 
 

## Solution
print logs to be seen during GH Actions

### CODE REVIEW

File: t/t.go [hunk 1]
```diff
@@ -168,6 +168,7 @@ func outputLogs(prefix string) {
 		out, err := logCmd.CombinedOutput()
 		x.Check(err)
 		f.Write(out)
+		fmt.Println(string(out))
 		// fmt.Printf("Docker logs for %s is %s with error %+v ", c.ID, string(out), err)
```
Comment (skrdgraph) [inline t/t.go:172]: open line #172, and rm line#171

Comment (skrdgraph) [reply t/t.go]: pls verify the CI run logs to make sure you have the right log statements

Comment (dshekhar95) [reply t/t.go]: will have to wait for a failure, 172 opened

**Comment by skrdgraph** [comment]
Closing this out because the issue is now identified & fixed.

**Pull request closed without merging**